src/content/docs/identityserver/ui/login/dynamicproviders.md

```diff
@@ -81,40 +158,65 @@ var dynamicSchemes = (await _identityProviderStore.GetAllSchemeNamesAsync())
 providers.AddRange(dynamicSchemes);
 ```
 
-### Store And Configuration Data
-
-To use the dynamic providers feature an [identity provider store](/identityserver/reference/stores/idp-store/) must be
-provided that will load [model data](/identityserver/reference/models/idp/) for the OIDC identity provider to be used.
-If you're using the [Entity Framework Integration](/identityserver/data/ef/) then this is implemented for you.
+The above code will query the identity provider store for all statically registered authentication schemes and merge them with (enabled) dynamic providers.
 
 :::note
-Like other configuration data in IdentityServer, by default the dynamic provider configuration is loaded from the store
-on every request unless caching is enabled.
-If you use a custom store, there is
-an [extension method to enable caching](/identityserver/data/configuration#caching-configuration-data).
-If you use the EF stores, there is general
-helper [to enable caching for all configuration data](/identityserver/data/ef#enabling-caching-for-configuration-store).
+The dynamic identity provider store API is deliberately separate from the `IAuthenticationSchemeProvider` provided by ASP.NET Core, which returns the
+list of statically configured providers (from `Startup.cs`).
+
+This split allows the developer to have more control over the customization on the login page. For example, there might be hundreds or
+thousands on dynamic providers, and therefore you would not want them displayed on the login page. At the same time, you might have a
+few social providers statically configured that you would want to display.
 :::
 
-The configuration data for the OIDC provider is used to assign the configuration on the ASP.NET
-Core [OpenID Connect Options](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.authentication.openidconnect.openidconnectoptions)
-class, much like you would if you were to statically configure the options when using `AddOpenIdConnect()`.
-The [identity provider model documentation](/identityserver/reference/models/idp) provides details for the model
-properties and how they are mapped to the options.
+## Callback Paths
+
+As part of the architecture of the dynamic providers feature, different callback paths are required and are
+automatically set to follow a convention. The convention of these paths follows the form of `~/federation/{scheme}/{suffix}`.
```

Comment:
I would add a tip here: Even if you don't use dynamic providers yet, it's a good idea to adopt this pattern for the callback paths to enable a smooth transition if you ever migrate to dynamic providers.